controllers/cluster/Makefile

```diff
@@ -62,7 +62,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 
 .PHONY: build
 build:  ## Build manager binary.
-	CGO_ENABLED=0 GOOS=linux go build -o bin/manager main.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/controller-cluster-amd64 main.go
```

Comment:
There is something wrong with writing like this, which will affect the CI of the cluster image.